experiments/AMIP/moist_mpi_earth/coupler_utils/general_helper.jl

```diff
@@ -1,9 +1,10 @@
 # most of these functions are temporary helpers until upstream issues are resolved
 
 # TODO: unify with coupler interface
-struct CouplerSimulation{I, F, D, B, T, M, P}
+struct CouplerSimulation{I, F, S, D, B, T, M, P}
     Δt::I
     t::F
+    tspan::S
```

Comment:
Depending on the type of `tspan`, it may not be necessary to also store `Δt`. For example, if `tspan = tstart:Δt:tstop` then `Δt = tspan.step`.

Reply:
That would be a neat solution! :) I was following the notation of `ClimaAtmos` (where they use `tspan` to set the `ODEProblem`), and I believe Kat did the same for the Bucket. Maybe to be consistent for now, we can suggest this as part of the holistic interface revamp when we explore Dennis's design?